dir.props

```diff
@@ -104,7 +104,7 @@
     <DnuRestoreCommand>$(DnuRestoreCommand) restore</DnuRestoreCommand>
     <DnuRestoreCommand Condition="'$(ParallelRestore)'=='true'">$(DnuRestoreCommand) --parallel</DnuRestoreCommand>
     <DnuRestoreCommand Condition="'$(UseNuGetHttpCache)'!='true'">$(DnuRestoreCommand) --no-cache</DnuRestoreCommand>
-    <DnuRestoreCommand>$(DnuRestoreCommand) --packages "$(PackagesDir.TrimEnd('/\'.ToCharArray()))" $(DnuRestoreSource)</DnuRestoreCommand>
+    <DnuRestoreCommand>$(DnuRestoreCommand) --packages "$(PackagesDir.TrimEnd('/').TrimEnd('\'))" $(DnuRestoreSource)</DnuRestoreCommand>
```

Comment:
There was some breaking changes in MSBuild with netcoreapp2.0 caused by the new overload.  See [https://github.com/Microsoft/msbuild/issues/1634](https://github.com/Microsoft/msbuild/issues/1634).

/cc @rainersigwald @AndyGerlicher - in case they want to know about this change for netcoreapp2.0.  It feels like a bug.

---
In reply to: [110289282](https://github.com/dotnet/corefx/pull/18035#discussion_r110289282) [](ancestors = 110289282)